src/styles/homeForm.scss

```diff
@@ -0,0 +1,32 @@
+$backColor: #0f0f0f;
+$baseColor: #FFFFFF;
+
+.header {
+  position: fixed;
+  width: 100%;
+  
+}
+
+.site-layout {
+    background: #fff;
+    padding: 0 50px !important;
+    margin-top: 64px !important;
+}
+
+.breadcrumb{
+    margin: 20px 0 !important;
+}
+.site-layout-background {
+    padding: 24px; 
+    min-height: 500px; 
+    background: $baseColor;
+}
```

Comment:
you can place this selector as a part of .site-layout selector. just use `&-background` inside .site-layout selector. Pls read scss tips and docs